test/versioned/express-esm/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "express-esm-tests",
+  "version": "0.0.0",
+  "type": "module",
+  "private": true,
+  "engines": {
+    "node": ">=14"
```

Comment:
Do we need this `engines` if we're using the one inside of the `tests` stanza?

Reply:
this one is for linting.  there's probably something specific in here but i can verify.  I just copied this from `test/versioned/express` so may not be necessary